benchmark_render.rb

```diff
@@ -75,11 +79,29 @@ def nested_collection
 
   def vc_loop
   end
+
+  def phlex_loop
+    render PhlexLoop.new
+  end
```

Comment:
This is the canonical way to use Phlex — it's not just for partials, but the entire view layer. It might be interesting to have an example of Phlex used just for the inner component rendered inside an ERB view though.